src/Blockcore.Explorer/ClientApp/src/app/explorer/contractaddress/contractaddress.component.html

```diff
@@ -0,0 +1,106 @@
+<app-search></app-search>
```

Comment:
One question I was not sure about is should this move in to its own folder for example `explorer/cirrus/xxx` 
For now I will leave it as is